code/datums/manufacturing.dm

```diff
@@ -1935,6 +1935,7 @@ proc/get_nice_mat_name_for_manufacturers(mat)
 /datum/manufacture/hat_ltophat
 	name = "Large Top Hat"
 	item_paths = list("FAB-1")
+	item_names = list("Fabric")
```

Comment:
```suggestion
```
As I mentioned the Fabric item name description is unnecessary now as it's generated automatically.